cockpit/src/subscriptions-client.js

```diff
@@ -25,7 +25,7 @@ const attachService = service.proxy('com.redhat.RHSM1.Attach', '/com/redhat/RHSM
 const entitlementService = service.proxy('com.redhat.RHSM1.Entitlement', '/com/redhat/RHSM1/Entitlement');
 const unregisterService = service.proxy('com.redhat.RHSM1.Unregister', '/com/redhat/RHSM1/Unregister');
 const productsService = service.proxy('com.redhat.RHSM1.Products', '/com/redhat/RHSM1/Products');
-const legacyService = cockpit.dbus('com.redhat.SubscriptionManager');  // FIXME replace?
+const consumerService = service.proxy('com.redhat.RHSM1.Consumer', '/com/redhat/RHSM1/Consumer');  // FIXME replace?
```

Comment:
Is the comment "FIXME replace?" still valid?

Reply:
Nope, not valid, I will take that out